### PR TITLE
Fix: ajusted -> adjust

### PR DESCRIPTION
**Pull request by Gootector**
Fix: ajusted -> adjust

Greetings,
Gootector

Resolves: #NNNNN 

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

**Comment by Gootector** [comment]
@cbjeukendrup One more typo.

File: src/importexport/mei/internal/meiconverter.cpp [hunk 1]
```diff
@@ -2432,7 +2432,7 @@ std::pair<libmei::Note, libmei::Accid> Convert::pitchToMEI(const engraving::Note
     int alterInt  = tpc2alterByKey(pitch.tpc2, engraving::Key::C);
 
     // @oct
-    // We need to ajusted the pitch to its transposed value for the octave calculation
+    // We need to adjusted the pitch to its transposed value for the octave calculation
```
Comment (cbjeukendrup) [inline src/importexport/mei/internal/meiconverter.cpp:2435]: It looks like it should actually be `adjust`, doesn't it?

Comment (Gootector) [reply src/importexport/mei/internal/meiconverter.cpp]: Of course - "to adjust".

**Review by cbjeukendrup — APPROVED** [review]
Thanks!

**Pull request merged**